File: backend/src/data/seed-data.ts
export const defaultOperators = [
  { name: "Unimed" },
  { name: "Camperj" }
];

export const defaultIndicators = [
  { name: "01- Indicador de Fluxo Assistencial", targetType: "PERCENTAGE", targetDirection: "HIGHER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE" },
  { name: "02- Nº de Intercorrências", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE" },
  { name: "03- Taxa de Internação Hospitalar", targetType: "PERCENTAGE", targetDirection: "LOWER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE" },
  { name: "04- Nº de óbitos", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE" },
  { name: "05- Taxa de Alterações de PAD", targetType: "PERCENTAGE", targetDirection: "LOWER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE" },
  { name: "06- Quantitativo de pacientes AD e ID", targetType: "NUMERIC", targetDirection: "HIGHER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE" },
  { name: "07- Nº de pacientes infectados", targetType: "PERCENTAGE", targetDirection: "LOWER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE", observations: "INÍCIO DE USO DE ANTIBIÓTICO EM 48H, META 0%" },
  { name: "08- Nº de eventos adversos", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE" },
  { name: "09- Nº de ouvidorias", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0, comparisonInterval: "ABSOLUTE" }
];

export const defaultSubindicatorsList = [
  { name: "1.1- Alta Domiciliar", parentPrefix: "01-", targetType: "PERCENTAGE", targetDirection: "HIGHER_BETTER", targetValue: 5 },
  { name: "1.2- Admissão", parentPrefix: "01-", targetType: "NUMERIC", targetDirection: "HIGHER_BETTER", targetValue: 0 },
  { name: "2.1- Resolvidas em domicílio", parentPrefix: "02-", targetType: "NUMERIC", targetDirection: "HIGHER_BETTER", targetValue: 0 },
  { name: "2.2- Necessidade de Remoção APH", parentPrefix: "02-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "3.1- Deterioração clínica", parentPrefix: "03-", targetType: "PERCENTAGE", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "3.2- Não aderência ao tratamento", parentPrefix: "03-", targetType: "PERCENTAGE", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "4.1- Menos de 48 horas após implantação", parentPrefix: "04-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "4.2- Mais de 48 horas de implantação", parentPrefix: "04-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "5.1- ↑ PAD", parentPrefix: "05-", targetType: "PERCENTAGE", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "5.2- ↓ PAD", parentPrefix: "05-", targetType: "PERCENTAGE", targetDirection: "HIGHER_BETTER", targetValue: 10 },
  { name: "8.1- Quedas", parentPrefix: "08-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "8.2 - Broncoaspiração", parentPrefix: "08-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "8.3 - Lesão por pressão", parentPrefix: "08-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "8.4 - Decanulação", parentPrefix: "08-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "8.5- Saida acidental da GTT", parentPrefix: "08-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "7.1- <48h Início de Antibiótico", parentPrefix: "07-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "7.2- >48h Pós-Antibiótico", parentPrefix: "07-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 },
  { name: "9.1- Elogios", parentPrefix: "09-", targetType: "NUMERIC", targetDirection: "HIGHER_BETTER", targetValue: 0 },
  { name: "9.2- Sugestões", parentPrefix: "09-", targetType: "NUMERIC", targetDirection: "HIGHER_BETTER", targetValue: 0 },
  { name: "9.3- Reclamações e Solicitações", parentPrefix: "09-", targetType: "NUMERIC", targetDirection: "LOWER_BETTER", targetValue: 0 }
];

export const camperjPatients = [
  "ALTIR DA ROCHA CUNHA", "ALVARO JOSE DA SILVA MELO", "ANA CAROLINA MENDES NOGUEIRA GOMES",
  "ANGELA MARIA MENEZES DE LIMA", "ANIBAL GONCALVES ALVES", "BENEDIR DE OLIVEIRA GASPAR",
  "DAISY AMOEDO BARREIRA", "DANILO DOMINGUES DE CARVALHO FILHO", "DEA ARAUJO DE AZEVEDO",
  "EDISON SANTOS CORREA", "EDUARDO CARLOS CARDOSO", "ELZA GUDULA MARIA DELBAERE",
  "EVANDRO DE OLIVEIRA SILVA", "FABIO CAVALCANTI DE ALBUQUERQUE MASSA", "FERNANDO CABRAL GOMES",
  "FERNANDO LÚCIO LAGOEIRO GUIMARÃES ABREU", "GENEIR HOLOSBACK ROSSES", "HUGO GOLDEMBERG",
  "IDE CORREA CARDOSO", "IZA MARIA BASTOS VIANNA", "JANAINA MARQUES CORREA MELO",
  "JOAO BOSCO FLEURY", "KATIA COSTA MARQUES DE FARIA", "LAIS FERNANDES GARCIA",
  "LEONEL DOS SANTOS", "LUCIA HELENA DOS SANTOS LUSQUINOS RODRIGUES", "LUCIA NEVES OLIVEIRA",
  "LUIZ FERNANDO DA COSTA GOMES", "LUIZ MARIO ROSSES FILHO", "MARCIA LEITE DE ABREU",
  "MARIA CRISTINA DA SILVA GAERTNER", "MARIA EDUARDA PACHECO NEVARES ALVES", "MARIA ELIANE GABRIEL ENNE",
  "MARIA GONCALVES SOARES", "MARILEA BARROSO MAGALHAES", "MARINA BRASIL FREITAS",
  "NEIDE RODRIGUES", "NILDA MARIA BENEVIDES DE MIRANDA", "NILZA COELHO BORDALLO",
  "ODAIA ALVES DOS REIS", "OZELINA DE SOUZA BARROS", "PAULO OSCAR DE FARIA",
  "RAMONA VIRGINIA ZACARIAS DE BECKERIG", "REGINA COELI TOSTES CALDAS", "RICARDO LUIZ DA SILVA REIS",
  "RUBINEIDE CAMPOS DOS REIS", "RUTE ELENA FERREIRA GOMES", "VERA LUCIA FERREIRA TAROUQUELA",
  "VERA MARIA FLORENCIO BERTO", "WANDA DE ALMEIDA VACITE", "ZENEIDE GAUDIE LEY LAGOEIRO DE MAGALHAES"
];

export const unimedPatients = [
  "BEATRIZ RIBEIRO ALVES", "CLARISSE JEANNE OLIVEIRA DOS SANTOS", "CLEUZA TEIXEIRA DE ARAUJO",
  "DENIZAR QUEIROZ", "DENIZE DE SOUZA BOM", "EMANUEL SOUZA MAIA", "GABRIELE CARDOSO MORAES",
  "GEANETTE DOROTHEIA PIMENTEL DA SILVA", "GILDA HELOISA DE SOUZA LIMA", "ISABELLE OLIVEIRA ROSA",
  "ITAMAR CHAVES DURAO", "JOAO LACERDA PEREIRA", "JOAQUIM PAULO MATOS", "JOE DAMIAO SESTELLO QUADROS",
  "JONAIR MACHADO DE ABREU", "JONAS MARIANO DE ALMEIDA", "JOSE ABELARDO SOUZA ANDRADE",
  "JOSE CARLOS DE LIMA", "LENICE SALLES PEREIRA", "LUCILIA DE CARVALHO GIMENEZ",
  "LUCIO DA SILVA RIBEIRO", "LUISA DO NASCIMENTO ELOI", "MARCIO LUIZ PISANI DE SOUZA",
  "MARIA ALICE SANTOS", "MARIA BATISTA DA SILVA LIRA", "MARIA DA GLÓRIA TOBIAS BARBOSA",
  "MARIA DA PENHA OLIVEIRA D ALMEIDA", "MARIA DAS GRACAS DOS SANTOS SILVA", "MARIA DE LOURDES DA SILVA SEMEDO",
  "MARIA DO CARMO LOURENCO DA SILVA", "MARIA LUIZA SOUZA NUNES", "MARIA ROSA DE JESUS",
  "MARINA FELIX DE SOUZA", "NELLY DE SOUZA BELEM RIBEIRO", "PEDRO PACHECO DE MATTOS",
  "REBECCA SIQUEIRA ROSA", "SIMONE DIAS LOPES", "WALDIR NUNES RIBEIRO"
];

File: backend/seed_data.py
default_operators = [
    {"name": "Unimed"},
    {"name": "Camperj"}
]

default_indicators = [
  {
    "name": "01 - Indicador de Fluxo Assistencial",
    "targetType": "PERCENTUAL",
    "targetDirection": "MAIOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "subindicators": [
      { "name": "1.1 - Alta Domiciliar", "targetType": "PERCENTUAL", "targetDirection": "MAIOR", "targetValue": 5 },
      { "name": "1.2 - Admissão", "targetType": "NÚMERICO", "targetDirection": "MAIOR", "targetValue": 0 }
    ]
  },
  {
    "name": "02 - Nº de Intercorrências",
    "targetType": "NÚMERICO",
    "targetDirection": "MENOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "subindicators": [
      { "name": "2.1 - Resolvidas em domicílio", "targetType": "NÚMERICO", "targetDirection": "MAIOR", "targetValue": 0 },
      { "name": "2.2 - Necessidade de Remoção APH", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 }
    ]
  },
  {
    "name": "03 - Taxa de Internação Hospitalar",
    "targetType": "PERCENTUAL",
    "targetDirection": "MENOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "subindicators": [
      { "name": "3.1 - Deterioração clínica", "targetType": "PERCENTUAL", "targetDirection": "MENOR", "targetValue": 0 },
      { "name": "3.2 - Não aderência ao tratamento", "targetType": "PERCENTUAL", "targetDirection": "MENOR", "targetValue": 0 }
    ]
  },
  {
    "name": "04 - Nº de óbitos",
    "targetType": "NÚMERICO",
    "targetDirection": "MENOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "subindicators": [
      { "name": "4.1 - Menos de 48 horas após implantação", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 },
      { "name": "4.2 - Mais de 48 horas de implantação", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 }
    ]
  },
  {
    "name": "05 - Taxa de Alterações de PAD",
    "targetType": "PERCENTUAL",
    "targetDirection": "MENOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "subindicators": [
      { "name": "5.1 - ↑ PAD", "targetType": "PERCENTUAL", "targetDirection": "MENOR", "targetValue": 0 },
      { "name": "5.2 - ↓ PAD", "targetType": "PERCENTUAL", "targetDirection": "MAIOR", "targetValue": 10 }
    ]
  },
  {
    "name": "06 - Quantitativo de pacientes AD e ID",
    "targetType": "NÚMERICO",
    "targetDirection": "MAIOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "subindicators": [
      { "name": "6.1 - AD (Atenção Domiciliar)", "targetType": "NÚMERICO", "targetDirection": "MAIOR", "targetValue": 0 },
      { "name": "6.2 - ID (Internação Domiciliar)", "targetType": "NÚMERICO", "targetDirection": "MAIOR", "targetValue": 0 }
    ]
  },
  {
    "name": "07 - Nº de pacientes infectados",
    "targetType": "PERCENTUAL",
    "targetDirection": "MENOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "observations": "INÍCIO DE USO DE ANTIBIÓTICO EM 48H, META 0%",
    "subindicators": [
      { "name": "7.1 - <48h Início de Antibiótico", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 },
      { "name": "7.2 - >48h Pós-Antibiótico", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 }
    ]
  },
  {
    "name": "08 - Nº de eventos adversos",
    "targetType": "NÚMERICO",
    "targetDirection": "MENOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "subindicators": [
      { "name": "8.1 - Quedas", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 },
      { "name": "8.2 - Broncoaspiração", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 },
      { "name": "8.3 - Lesão por pressão", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 },
      { "name": "8.4 - Decanulação", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 },
      { "name": "8.5 - Saída acidental da GTT", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 }
    ]
  },
  {
    "name": "09 - Nº de ouvidorias",
    "targetType": "NÚMERICO",
    "targetDirection": "MENOR",
    "targetValue": 0,
    "comparisonInterval": "ABSOLUTO",
    "subindicators": [
      { "name": "9.1 - Elogios", "targetType": "NÚMERICO", "targetDirection": "MAIOR", "targetValue": 0 },
      { "name": "9.2 - Sugestões", "targetType": "NÚMERICO", "targetDirection": "MAIOR", "targetValue": 0 },
      { "name": "9.3 - Reclamações e Solicitações", "targetType": "NÚMERICO", "targetDirection": "MENOR", "targetValue": 0 }
    ]
  }
]

camperj_patients = [
    "AGRIPINA CUNHA MONTEIRO", "ALBERTO JORGE DE OLIVEIRA PESTANA", "ALTAIR RAMOS DA SILVA CORREA", "ANGELICA DIAS NUNES", "ANTONIO DE PAIUA",
    "ANTONIO REIS", "ARCIDIO GONCALVES PEREIRA", "CEZAR CORREIA MACHADO", "DÉCIO SILVA ALVES DE OLIVEIRA", "DELZA GOMES DA SILVA ROCHA",
    "DEUSA CORTEZ PEREIRA", "DJALMA FIALHO DOS SANTOS", "ELISABETH MONTEIRO CARNEIRO", "ELZA GOMES RODRIGUES", "ETELVINA GONÇALVES MACHADO CORDEIRO",
    "EVANIR VIEIRA ROQUE", "FRANCISCO SANTIAGO BICHARA", "GENOVEVA LOUREIRO MARTINS", "IBAIR ALVES SILVA", "IRACI MONTEIRO DE ABREU",
    "IRAILDES BRAGA MAIA", "JANDIRA CINTRA ALMEIDA", "JAYME MOREIRA JUNIOR", "JAYR DE CASTRO NUNES", "JOSÉ AUGUSTO BASTOS MACEDO",
    "JOSE EDUARDO BROM", "JOSÉ ROBERTO DE MENDONÇA BARRETO", "JOSE ROBERTO FARIAS DA SILVA", "JUCELINA BARBOZA", "LAZARA PEREIRA FURLAN",
    "LINDOLFO JOSE DE ALMEIDA FILHO", "LUDMILLA COSTA DE ALMEIDA", "LUIZ ROBERTO BARROS OLYMPIO", "MARIA CANDIDA MENDES FERNANDES",
    "MARIA CILENE BARRETO DOS SANTOS", "MARIA DE LOURDES GUIMARAES FERREIRA", "MARIA DE LOURDES ROSA", "MARIA ISABEL MARINS BASTOS DIAS",
    "MARIA LUIZA TAVARES", "MARIA NIDIA CARVALHAES GUIMARÃES", "MARLENE TEIXEIRA DA SILVA", "MURILLO BAPTISTA JUNIOR", "NATHALIA TIBURCIO PINHEIRO",
    "NEIDA SANTOS ALMEIDA", "NELY MONTEIRO DE CARVALHO", "NEUSA ROCHA ALVES", "NIVAL DA SILVA NEVES", "NYCIA BRUNO DO CARMO", "RAPHAEL CORREA ROCHA",
    "SAHADE JORGE CHERFAN", "SEBASTIÃO CORDEIRO DE MORAES", "SOLANGE BUSTAMANTE BORGES", "UBIRATAN FERNANDES DE CARVALHO", "VANTUIR FERNANDES",
    "WALDIR NUNES RIBEIRO", "ZARIFE RANGEL DE CASTRO", "ZELIA DE SOUSA E SILVA COELHO"
]

unimed_patients = [
    "ADA JARDIM MACHADO", "ALBERTINA LOURENCO ALVES", "ALBINA DA SILVA PEREIRA TANNUS", "ALDA VIEIRA BRANDAO", "ANADIA MOREIRA AMARAL",
    "ANGELICA REIS RAMOS PEREIRA", "ANTONIO PEREZ PEREZ", "DANIEL VIEIRA BARBOSA", "DELAIR MACHADO", "EDELZUITE PEREIRA BARRETO",
    "ENY MARTINS VILLELA", "EULER TEIXEIRA VILLELA", "GLAUCE DE CASTRO PEIXOTO RODRIGUES", "HELIODORO NATIVIDAD CASTILLO CASTILLO",
    "ILCA COIMBRA MAZARINI", "ITALIA REGINA CARNEIRO PAIS", "LEILA BRAGA", "LEONILA GOMES BAPTISTA", "LIGIA RODRIGUES BAPTISTA",
    "MARIA BATISTA DE ARAÚJO", "MARIA DA PENHA BRAGA DA CUNHA", "MARIA DE LOURDES DE AZEVEDO CINTRA", "MARIA EDNEA CHAGAS DE SOUSA",
    "MARIA ENIL VIEIRA DE CASTRO MACHADO", "MARIA JOSE FERREIRA CAMPOS", "MARINA DA SILVA SOARES", "NILMA MARINS TAVARES",
    "RUTH REZENDE BICALHO", "SHIRLEY DUARTE GUIMARÃES", "SONIA DE SOUSA BARRETO CANGUSSU", "WALDINEA GOMES RANGEL NUNES", "ZILDA FERNANDES FREITAS"
]
